source/jormungandr/jormungandr/realtime_schedule/tests/sytral_test.py

```diff
@@ -36,34 +36,42 @@
 import pytest
 
 
-def make_url_test():
+def make_url_params_test():
     sytral = Sytral(id='tata', service_url='http://bob.com/')
 
-    url = sytral._make_url(MockRoutePoint(line_code='line_toto', stop_id='stop_tutu'))
+    # The return is like [(stop_id, val1), (stop_id, val2), (direction_type, val), ...]
+    params = sytral._make_params(MockRoutePoint(line_code='line_toto', stop_id='stop_tutu'))
 
-    # it should be a valid url
-    assert validators.url(url)
+    assert len(params) == 1
+    assert 'stop_tutu' in params[0]
```

Comment:
Is it possible to test it like this? (to test the stop's id is in "stop_id" param)
```suggestion
    assert params == [("stop_id", "stop_tutu")]
```
or
```suggestion
    assert params[0] == ("stop_id", "stop_tutu")
```
Same below if relevant.

Reply:
I prefer your first proposition. 
Thanks for the trick :ok_hand: